inc/translations.php

```diff
@@ -75,38 +69,58 @@ private function get_locale_data() {
 		 *
 		 * @since 2.3.5
 		 *
-		 * @param $locales
+		 * @param array $locales All locale info for AIOSEOP from translate.wordpress.org.
+		 * @var array $locale Individual locale info for AIOSEOP from translate.wordpress.org.
+		 * @var string $wp_locale Locale name from translate.wordpress.org (does not include formal designation).
+		 * @var string $formal Indication of whether currently active locale is formal.
+		 * @var string $slug Indication of whether locale from translate.wordpress.org is formal.
+		 * @var string $current_locale Currently active locale.
 		 */
 		private function set_current_locale_data( $locales ) {
 
+			$current_locale = $this->current_locale;
+
+			if ( strpos( $current_locale, '_formal' ) ) {
+				$formal = $this->formal = 'formal';
```

Comment:
This is being flagged as `phpcs Squiz.PHP.DisallowMultipleAssignments.Found`, and is also one of the current excluded rules.